### DISCUSSION TITLE
Make "open in new tab" an optional flag?

### DISCUSSION THREAD

**Discussion by rei2work** [tadashi-aikawa/obsidian-another-quick-switcher#243] [Ideas]
I am using the Recent search window, I am looking for an option to open the file in new tab; I do notice that  [Ctrl enter] does "Open in new tab".

I think opening files in a new tab is kind of a default behavior in other editors but I found no way to configure it into a default in Obsidian. Just would like this to be configurable so I don't even need to use  [Ctrl enter] to do so. (I actually have  [Ctrl enter] globally mapped to do other things with AHK lol). 

Much Thanks!!

**Comment by tadashi-aikawa** [comment]
@rei2work 
How about setting 'Enter' to 'open in new tab' for 'Hot keys in dialog'? (At the same time, it will be necessary to set 'open' to something other than 'Enter'.)

![image](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/9500018/18be1b7e-1a22-4b82-a52e-19c2202a7532)
